src/Umbraco.Web/Editors/ContentController.cs

```diff
@@ -347,6 +347,9 @@ public ContentItemDisplay GetEmpty(string contentTypeAlias, int parentId)
 
             var emptyContent = Services.ContentService.CreateContent("", parentId, contentType.Alias, UmbracoUser.Id);
             var mapped = AutoMapperExtensions.MapWithUmbracoContext<IContent, ContentItemDisplay>(emptyContent, UmbracoContext);
+            // translate the content type name if applicable
+            mapped.ContentTypeName = Services.TextService.UmbracoDictionaryTranslate(mapped.ContentTypeName);

```

Comment:
Hi @kjac 

Thanks for this PR. I had this on my todo list as well. So now I can check it off :-)

Maybe a check can be done first if the name starts with a # so we only translate it when it is actually a dictionary ?

Dave [Community Pull Request team](https://umbraco.com/blog/introducing-the-new-cpr-team/)

Reply:
Didn't know that. 

So code looks good. Need to do a test before I can approve